gapii/client/capture.go

```diff
@@ -68,6 +66,14 @@ const (
 	GvrAPI = uint32(1 << 3)
 )
 
+const (
+	messageData       byte = 0x00
+	messageStartTrace byte = 0x01
+	messageEndTrace   byte = 0x02
+	messageError      byte = 0x03
+	messageInvalid    byte = 0xff
+)
```

Comment:
Please make these typed so that we know what they are. I.e.
```
type messageType byte
const(
    messageData messageType = 0x00
    ...
)
```

Reply:
Done